packages/esm-ward-app/src/ward-view/ward-view.test.tsx

```diff
@@ -20,10 +27,15 @@ jest.mocked(useSession).mockReturnValue({
   sessionId: 'sessionId',
 });
 
+const mockFeatureFlag: jest.Mock = jest.fn(() => true);
+
 jest.mock('@openmrs/esm-framework', () => {
   return {
     ...jest.requireActual('@openmrs/esm-framework'),
     useLocations: jest.fn().mockImplementation(() => mockLocations.data.results),
+    useFeatureFlag: () => {
```

Comment:
We can remove this if we do what I suggested above.

(We can't do the same thing for `useLocations` because it's not defined here: https://github.com/openmrs/openmrs-esm-core/blob/2fb9ee778f3b3d4b7d5dd7ad4b214fa1b60799b1/packages/framework/esm-react-utils/mock.tsx)